SC_redesign/Crop_and_Soil/field/field.py

```diff
@@ -24,7 +26,13 @@
 class Field:
     """object representing an agricultural field"""
 
-    def __init__(self, field_data: Optional[FieldData] = None, soil: Optional[Soil] = None):
+    def __init__(self, tillage_events: Optional[TillageEvent] = None, field_data: Optional[FieldData] = None,
+                 soil: Optional[Soil] = None, ):
```

Comment:
```suggestion
    def __init__(self, field_data: Optional[FieldData] = None, soil: Optional[Soil] = None, 
                 tillage_events: Optional[TillageEvent] = None):
```
Adding to the initialization parameters this way ensures compatibility with any old code that creates field instances with calls like
```python
test_field = Field(field_data, soil)
```